Data/ByteString/Char8.hs

```diff
@@ -897,6 +897,8 @@ unwords = intercalate (singleton ' ')
 -- | readInt reads an Int from the beginning of the ByteString.  If there is no
 -- integer at the beginning of the string, it returns Nothing, otherwise
 -- it just returns the int read, and the rest of the string.
+--
+-- /Note/: This function will overflow the Int for large integers.
```

Comment:
Since the other `Note:`s throughout the Haddocks don't use any markup, please do not use `/Note/`. If anything I use  boldface markup for this, i.e. `_Note_`, for packages I maintain. But this would deserve a separate commit.

Reply:
OK, done!